python/ray/tests/test_metrics.py

```diff
@@ -133,6 +136,52 @@ def test_prometheus_endpoint():
         )
 
 
+_test_counter = 0
+
+
+def test_metrics_helper():
```

Comment:
Can you add e2e tests for new metrics?

Reply:
hmm why won't unit test suffice here? :)